src/SFA.DAS.ApprenticeFeedback.Application.UnitTests/Queries/WhenRequestingFeedbackTargetsForUpdate.cs

```diff
@@ -0,0 +1,437 @@
+                StartDate = DateTime.UtcNow.AddMonths(-6),
+                Status = (int)Domain.Models.Enums.FeedbackTargetStatus.Active,
+                EligibilityCalculationDate = DateTime.UtcNow.AddDays(-2),
+            };
+        }
+
+        private static Domain.Entities.ApprenticeFeedbackTarget AFT_NoFeedback(string providerName)
+        {
+            return new Domain.Entities.ApprenticeFeedbackTarget()
+            {
+                ProviderName = providerName,
+                StartDate = DateTime.UtcNow.AddMonths(-6),
+                Status = (int)Domain.Models.Enums.FeedbackTargetStatus.Active,
+                EligibilityCalculationDate = DateTime.UtcNow.AddDays(-100),
+            };
+        }
+
+        private static Domain.Entities.ApprenticeFeedbackTarget AFT_FeedbackGivenRecently(string providerName)
+        {
+            return new Domain.Entities.ApprenticeFeedbackTarget()
+            {
+                ProviderName = providerName,
+                StartDate = DateTime.UtcNow.AddMonths(-6),
+                Status = (int)Domain.Models.Enums.FeedbackTargetStatus.Active,
+                EligibilityCalculationDate = DateTime.UtcNow.AddDays(-100),
+                ApprenticeFeedbackResults = new List<Domain.Entities.ApprenticeFeedbackResult>() 
+                { 
+                    new Domain.Entities.ApprenticeFeedbackResult()
+                    {
+                        DateTimeCompleted = DateTime.UtcNow.AddDays(-2)
+                    }
+                }
+            };
+        }
+
+        private static Domain.Entities.ApprenticeFeedbackTarget AFT_EligibleToGiveFeedback(string providerName)
+        {
+            return new Domain.Entities.ApprenticeFeedbackTarget()
+            {
+                ProviderName = providerName,
+                StartDate = DateTime.UtcNow.AddMonths(-6),
+                Status = (int)Domain.Models.Enums.FeedbackTargetStatus.Active,
+                EligibilityCalculationDate = DateTime.UtcNow.AddDays(-100),
+                ApprenticeFeedbackResults = new List<Domain.Entities.ApprenticeFeedbackResult>()
+                {
+                    new Domain.Entities.ApprenticeFeedbackResult()
+                    {
+                        DateTimeCompleted = DateTime.UtcNow.AddDays(-95)
+                    }
+                }
+            };
+        }
+
+        #endregion Test Apprentice Feedback Targets
+
+        #region IQueryableExtensionTests
+
+        public class IQueryableExtensionTests
+        {
+            public class HasStarted
+            {
+                [Test]
+                [AutoMoqData]
+                public void When_ApprenticeshipNotStarted_Then_ReturnEmpty(
+                    [Frozen(Matching.ImplementedInterfaces)] ApprenticeFeedbackDataContext context,
+                    UtcTimeProvider dateTimeHelper
+                    )
+                {
+                    context.Add(AFT_NotStarted("Provider Name"));
```

Comment:
I'd be tempted to add your eligible criteria to the collection as well, so that rather than saying targets should be empty, it contains one, which is the eligible one. Proving the test ignores what it should, but still allows the valid one